graylog2-web-interface/src/components/lookup-tables/Cache.jsx

```diff
@@ -26,7 +27,12 @@ const Cache = React.createClass({
     return (
       <Row className="content">
         <Col md={6}>
-          <h2>{cache.title} <small>({plugin.displayName})</small></h2>
+          <h2>
+            {cache.title}
+            <ContentPackMarker contentPack={cache.content_pack} marginLeft={5} />
```

Comment:
Related to above: could we give it a left and right sibling prop instead of padding with spaces and inline styles?